test/brsTypes/components/Timer.test.js

```diff
@@ -0,0 +1,24 @@
+const brs = require("brs");
+const { Timer } = brs.types;
+
+describe("Timer", () => {
+    describe("stringification", () => {
+        it("initializes a new Timer component", () => {
+            let timer = new Timer();
+
+            expect(timer.toString()).toEqual(
+                `<Component: roSGNode:Timer> =
+{
+    change: <UNINITIALIZED>
+    focusable: false
+    focusedchild: invalid
+    id: 
```

Comment:
🤔  Quick question -- `id` and `control` are both left blank -- how is this different than <UNINITIALIZED>?

Reply:
There are a couple of things going on here. Firstly, this technically shouldn't be blank, it's a string so it should be `""`. It's a minor bug in `brs`, and #416 is filed to address it.

The other thing happening is that `<UNINITIALIZED>`is the value you get when you add a field, but not a value for that field. In our current implementation, this is essentially what's happening:
```
myNode.addField("change", "function")
```
Note that we know `change` _exists_, but it doesn't have a value, so it's `UNINITIALIZED`. That being said, the fact that this is happening here is because we haven't implemented default values for complex types like `function`, `node`, `array`, etc. So in the future, `change` will actually get a default value instead of `UNINITIALIZED`. There's another open PR to partially address that.

Reply:
Ah, got it!  Thanks for the explanation!